temporal-sdk/src/main/java/io/temporal/internal/sync/LocalActivityExecutionContextImpl.java

```diff
@@ -72,6 +73,18 @@ public boolean isDoNotCompleteOnReturn() {
         "isDoNotCompleteOnReturn is not supported for local activities");
   }
 
+  @Override
+  public boolean isUseLocalManualCompletion() {
+    throw new UnsupportedOperationException(
+        "isUseLocalManualCompletion is not supported for local activities");
```

Comment:
Why? it makes perfect sense for local activities implemented asynchronously.

Reply:
We need to rethink our behavior for limiting local activity throughput as currently semaphore and setMaxConcurrentActivityExecutionSize is only applied to normal activities. This would be a bigger change, and I suggest keep it out of scope for this PR.

Reply:
Let's keep it out of the scope of this PR. 

I believe that async local activities can be an important performance optimization in some cases.